Discord/DiscordEvents/presenceUpdate.js

```diff
@@ -0,0 +1,13 @@
+module.exports = async function (
+  /** 'module:"discord.js".Presence */ oldPresence,
+  /** 'module:"discord.js".Presence */ newPresence,
+  watchedMembers
+) {
+  if(watchedMembers.includes(newPresence.user.tag)) {
+    if(!oldPresence || newPresence.status !== oldPresence.status) {
+      let toSend = ":rotating_light: L'utilisateur "+newPresence.user.tag+", qui est sous surveillance, est désormais à l'état "+newPresence.status+" :rotating_light:.";
```

Comment:
```suggestion
      const toSend = ":rotating_light: L'utilisateur "+newPresence.user.tag+", qui est sous surveillance, est désormais à l'état "+newPresence.status+" :rotating_light:.";
```

Check Eslint avec les règles AirBNB pour corriger automatiquement ce genre de problème
https://github.com/ungdev/UA-api/blob/feat/lots-of-things/.eslintrc.json

Reply:
fait